### PR TITLE
Update to README.md.

### PR DESCRIPTION
**Pull request by robbiethegeek-usds**

### CODE REVIEW

**Comment by robbiethegeek-usds** [comment]
Hey @raquelromano-gov added you since you are probably about to setup your env, so please let me know if this is an improvement.

File: README.md [hunk 1]
```diff
@@ -17,26 +17,26 @@ Vets-api requires:
...
-```
-
-For an example, see `application.yml.example`
 
 1. Install Redis (on mac): `brew install redis`
 1. Follow post install instructions
   - always have Redis running as service
   - manually launch Redis `redis-server /usr/local/etc/redis.conf`
-1. Set the environment variables above according to your Redis configuration
+3. Set the environment variables above according to your Redis configuration
```
Comment (raquelromano-gov) [inline README.md]: Do these env variables still need to be set? If not, we can delete this line as well. If so, some guidance about how to set them would be good.

Comment (robbiethegeek-usds) [inline README.md]: will remove the line since it is actually in the copying over of the application.yml.example previously

File: README.md [hunk 2]
```diff
@@ -17,26 +17,23 @@ Vets-api requires:
 1. Install Postgres (on Mac): `brew install postgres`
 1. Get the code: `git clone https://github.com/department-of-veterans-affairs/vets-api.git; git submodule init; git submodule update`
```
Comment (raquelromano-gov) [inline README.md]: While you're editing this, may want to add that you need to cd into vets-api before calling git submodule

File: README.md [hunk 3]
```diff
@@ -17,26 +17,23 @@ Vets-api requires:
...
 1. Install gem dependencies: `cd vets-api; bundle install`
+1. Create a application.yml `cat ./application.yml.example > ./application.yml`
```
Comment (raquelromano-gov) [inline README.md]: Just started going through this--there's no application.yml.example in this directory--did you mean to incluce it in this PR?

**Comment by robbiethegeek-usds** [comment]
Feedback addressed

File: README.md [hunk 4]
```diff
@@ -15,28 +15,25 @@ Vets-api requires:
 *Note*: rbenv will also provide additional installation instructions in the console output. Make sure to follow those too.
 1. Install Bundler to manage dependencies: `gem install bundler`
 1. Install Postgres (on Mac): `brew install postgres`
-1. Get the code: `git clone https://github.com/department-of-veterans-affairs/vets-api.git; git submodule init; git submodule update`
+1. Get the code: `git clone https://github.com/department-of-veterans-affairs/vets-api.git; cd vets-api; git submodule init; git submodule update`
 1. Install gem dependencies: `cd vets-api; bundle install`
```
Comment (raquelromano-gov) [inline README.md]: Can remove 'cd vets-api' from this line now that it's on the line above

File: README.md [hunk 5]
```diff
@@ -15,28 +15,25 @@ Vets-api requires:
...
+1. Create a application.yml `cat ../config/application.yml.example > ./config/application.yml`
```
Comment (raquelromano-gov) [inline README.md]: Current directory is vets-api, right? So
cat ./config/application.yml.example

File: README.md [hunk 6]
```diff
@@ -15,28 +15,25 @@ Vets-api requires:
...
+1. Setup localhost certificates / keys
+  - Create a hidden folder in home directory:  `mkdir ~/.certs`
+  - Copy the [certificate](https://github.com/department-of-veterans-affairs/vets.gov-team/blob/master/Products/Identity/Identity%20Discovery%202016/certificates/vetsgov-localhost.crt) to ~/.certs
+  - Copy the [key](https://github.com/department-of-veterans-affairs/vets.gov-team/blob/master/Products/Identity/Identity%20Discovery%202016/certificates/vetsgov-localhost.key) to ~/.certs
 
 ### Database Setup
 1. Start Postgres: `postgres -D /usr/local/var/postgres`
 1. Create dev database: `bundle exec rake db:setup`
 *Note*: This will not work until you set up the environment variables (see below).
```
Comment (raquelromano-gov) [inline README.md]: Do you know if this refers to the Redis environment variables? If they are no longer needed for this to work, then this line can go away as well.

**Comment by cvalarida** [comment]
A few things I found while setting it up for the first time (Ubuntu 16.04):

1. The ruby version required seems to be specifically 2.3.0, not 2.3.x
2. The local postgres user must have superuser permission
    1. At least under Ubuntu, when I installed postgres, it didn't make a postgres user for my linux user (if that makes sense), so I had to:
        ```
        $ sudo su - postgres
        $ psql
        > CREATE USER {unix username} WITH SUPERUSER;
        ```
3. The command to start the server manually is `bundle exec rails server`; the \` is just premature

Hope that helps.

**Comment by robbiethegeek-usds** [comment]
@cvalarida Can you distill down any Linux specific issues into a single markdown file and add and link from this doc

**Pull request merged**